sematic/api/endpoints/events.py

```diff
@@ -66,6 +66,16 @@ def broadcast_pipeline_update(
     return _call_broadcast_endpoint(url=url, json_payload=json_payload, user=user)
 
 
+def broadcast_job_update(
```

Comment:
Is this needed to refresh the resources panel on the Dashboard?
It's not called anywhere.

Reply:
Yes, that's what it's for